e2e_tests/e2e_test.py

```diff
@@ -13,11 +13,8 @@
 @pytest.fixture(scope='module')
 def test_app():
     # Create the new board & update the board id environment variable
-    trello_board = TrelloAPI().create_board("SeleniumTest")
-    os.environ['TRELLO_BOARD_ID'] = trello_board.board_id
-    # Use our test config instead of the 'real' version
-    file_path = find_dotenv('.env.test')
-    load_dotenv(file_path, override=True)
+    mongo_db = MongoWrapper().create_database("SeleniumTest")

```

Comment:
Running this locally fails due to the .env file not having been loaded in - I'd expect this to create the database after the `.env` variables were loaded (on docker this is fine as we set them in the container)

Reply:
Ahh yes, I only run this test on docker. Will fix, I assume if I document that this is only supported as a test through docker that will also be valid?